geth/transactions/notifications.go

```diff
@@ -80,8 +80,8 @@ func NotifyOnReturn(queuedTx *common.QueuedTx) {
 			ID:           string(queuedTx.ID),
 			Args:         queuedTx.Args,
 			MessageID:    common.MessageIDFromContext(queuedTx.Context),
-			ErrorMessage: queuedTx.Err.Error(),
-			ErrorCode:    strconv.Itoa(sendTransactionErrorCode(queuedTx.Err)),
+			ErrorMessage: err.Error(),
+			ErrorCode:    strconv.Itoa(sendTransactionErrorCode(err)),
```

Comment:
because it is expected on consumer side? i can make this conversion when marshaling though

Reply:
But on the consumer side, it can be an integer and it should be ok. Here, it will just convert `1` to `"1"`. I don't think it will make any difference to status-react.